### ISSUE TITLE
Data from simulink output is not available

### ISSUE THREAD

**Issue by moorepants** [csu-hmc/perturbed-data-paper#105]
Just making this issue to note what Sandy and I already discussed. We either need to commit this to the `processed-data` directory in this repo if the data file is "small" or host it on Zenodo for download.

To clarify: this is the desired commanded control inputs to the treadmill for the different belt speeds.

**Comment by spinningplates** [comment]
@moorepants Did I ever send you this file?

**Comment by moorepants** [comment]
Yes, you did. I need to add it to the data download. I'll do that when I work on issue #134.

**Comment by moorepants** [comment]
I'll handle this one, but if you can deal with the other matlab ones, that would be great.

**Comment by moorepants** [comment]
@spinningplates I have the files containing the commanded longitudinal perturbations but not the one for the lateral. Can you send that to me ASAP?

**Comment by moorepants** [comment]
@spinningplates 

I decided on this. I've created a new compressed file `perturbation-signals.tar.gz` that contains:
- `README.rst` to explain the contents of the file.
- `longitudinal-perturbation.txt` a csv file for the belt speeds
- `lateral-perturbation.txt` a csv file for the base lateral motion

The file will be included with the current zenodo download or be as a separate zenodo item (I'm getting help from zenodo tech support to figure out how to do the first).

Regardless, this will be unzipped and if the default config is used the directory structure from the root of this repository will be like this:

```
raw-data/
|
--> perturbation-signals/
     |
     ---> longitundinal-perturbations.txt
     |
     ---> lateral-perturbations.txt
```
So all of the matlab scripts need to be updated to load from these files. I would recommend using PR #150.

**Comment by moorepants** [comment]
Ok this is fixed in #155. You can update your m-files to reflect the new data sources.
